src/main/java/se/sundsvall/supportmanagement/integration/db/NotificationRepository.java

```diff
@@ -18,4 +20,6 @@ public interface NotificationRepository extends JpaRepository<NotificationEntity
 
 	List<NotificationEntity> findAllByNamespaceAndMunicipalityIdAndOwnerId(String namespace, String municipalityId, String ownerId);
 
+	Optional<NotificationEntity> findByNamespaceAndMunicipalityIdAndOwnerIdAndAcknowledgedAndErrandIdAndType(final String namespace, final String municipalityId, final String ownerId, @NotNull final boolean acknowledged, final String errandId, final String type);
```

Comment:
Regarding the `@NotNull`: 
It's a primitive boolean. Do you really need this constraint validation here?